tests/integration/world_io_test.py

```diff
@@ -21,3 +20,36 @@ def testLoadWorld(self):
     def testLoadInvalidWorld(self):
         with self.assertRaises(LoadWorldException):
             GraspitCommander.loadWorld("")
+
+    def testImportObstacle(self):
+        GraspitCommander.importObstacle("floor")
```

Comment:
Hi Jared,
getObstacle wasn't among the services implemented-- just getBody.  The only property that gives is the body's pose.  Think that's useful to check (it should be at the origin after import).